### PR TITLE
[TelegramBotInterface] Add restart command

### PR DESCRIPTION
**Pull request by valouvaliavlo**
Require : https://github.com/Drakkar-Software/OctoBot-Services/pull/109

### CODE REVIEW

File: Services/Interfaces/telegram_bot_interface/telegram_bot.py [hunk 1]
```diff
@@ -128,6 +130,13 @@ def command_start(update, _):
         elif TelegramBotInterface._is_authorized_chat(update):
             TelegramBotInterface._send_message(update, interfaces_bots.UNAUTHORIZED_USER_MESSAGE)
 
+    @staticmethod
+    def command_restart(update, _):
+        if TelegramBotInterface._is_valid_user(update):
+            TelegramBotInterface._send_message(update, "I'll come back!!!!!")
```
Comment (GuillaumeDSM) [inline Services/Interfaces/telegram_bot_interface/telegram_bot.py:136]:
```suggestion
            TelegramBotInterface._send_message(update, "I'll come back !")
```
😂

**Review by GuillaumeDSM — APPROVED** [review]
👍

File: Services/Interfaces/telegram_bot_interface/telegram_bot.py [hunk 2]
```diff
@@ -109,7 +110,8 @@ def command_help(update, _):
             message += "/refresh\_portfolio or /rpf : `Forces OctoBot's real trader portfolio refresh using exchange " \
                        "data. Should normally not be necessary.`" + interfaces_bots.EOL
             message += "/pause or /resume: `Pauses or resumes me.`" + interfaces_bots.EOL
-            message += "/stop: `Stops me.`" + interfaces_bots.EOL
+            message += "/restart: `Stops me.`" + interfaces_bots.EOL
```
Comment (Herklos) [inline Services/Interfaces/telegram_bot_interface/telegram_bot.py:113]: You should inverse the two messages 😂

Comment (valouvaliavlo) [reply Services/Interfaces/telegram_bot_interface/telegram_bot.py:113]: Ah yes !

**Review by Herklos — APPROVED** [review]

**Review by GuillaumeDSM — DISMISSED** [review]

File: Services/Interfaces/telegram_bot_interface/telegram_bot.py [hunk 3]
```diff
@@ -109,6 +110,7 @@ def command_help(update, _):
             message += "/refresh\_portfolio or /rpf : `Forces OctoBot's real trader portfolio refresh using exchange " \
                        "data. Should normally not be necessary.`" + interfaces_bots.EOL
             message += "/pause or /resume: `Pauses or resumes me.`" + interfaces_bots.EOL
+            message += "/restart: `Restart me.`" + interfaces_bots.EOL
```
Comment (GuillaumeDSM) [inline Services/Interfaces/telegram_bot_interface/telegram_bot.py:113]:
```suggestion
            message += "/restart: `Restarts me.`" + interfaces_bots.EOL
```

**Review by Herklos — APPROVED** [review]

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**